flexinput/src/main/java/com/lytefast/flexinput/fragment/FlexInputFragment.java

```diff
@@ -126,6 +126,9 @@ public void onItemUnselected(final Object item) {
           }
 
           private void updateUi() {
+            if (getView() == null) {
```

Comment:
Does this happen on UI thread? If not should we just hold onto the view ref so there's no race here?